### PR TITLE
Feat/basic faq

### PR DESCRIPTION
**Pull request by daimeng**
Pulling some basic info over

### CODE REVIEW

**Comment by daimeng** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>962aaed</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://zodiac-landing-580l2zbjj-daimeng.vercel.app'>https://zodiac-landing-580l2zbjj-daimeng.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/daimeng/zodiac-landing/FSSi7DQQPD8MwXnuLLU9otsuMRRC'>https://vercel.com/daimeng/zodiac-landing/FSSi7DQQPD8MwXnuLLU9otsuMRRC</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/daimeng/zodiac-landing/actions/runs/2708608940)

**Comment by daimeng** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>5f9177a</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://zodiac-landing-lm42nrskn-daimeng.vercel.app'>https://zodiac-landing-lm42nrskn-daimeng.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/daimeng/zodiac-landing/94SijNLFaJJynX8mUuusK8xsYhpE'>https://vercel.com/daimeng/zodiac-landing/94SijNLFaJJynX8mUuusK8xsYhpE</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/daimeng/zodiac-landing/actions/runs/2708624541)

File: pages/faq.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import styles from './faq.module.css'
+
```
Comment (kimchicoffee) [inline pages/faq.js:2]: importing styles from the empty file? I guess this pr is only for contents and style comes later?

Comment (daimeng) [reply pages/faq.js:2]: ah yeah, i will add some styling, but mostly for content

**Pull request merged**